application-workloads/azure-gamedev/gamedev-vm/nestedtemplates/gamedev-vm.bicep

```diff
@@ -621,3 +684,7 @@ resource virtualMachine_enableAAD 'Microsoft.Compute/virtualMachines/extensions@
     autoUpgradeMinorVersion: true
   }
 }
+
+output Host_Name string = (!empty(publicIpId) ? reference(publicIpId, '2021-03-01').dnsSettings.fqdn : '')
+output UserName string = adminName
+output IPAddress string = (!empty(publicIpId) ? publicIpId : '')
```

Comment:
this isn't an ipAddress it's a resourceId - what's the intent?

Reply:
should have returned ip address, I hope this is fixed via "publicIp.ipAddress"